test/Dialect/Seq/firreg.mlir

```diff
@@ -1,5 +1,5 @@
 // RUN: circt-opt %s -verify-diagnostics --lower-seq-to-sv | FileCheck %s --check-prefixes=CHECK,COMMON
-// RUN: circt-opt %s -verify-diagnostics --pass-pipeline="builtin.module(lower-seq-to-sv{disable-reg-randomization})" | FileCheck %s --check-prefix COMMON --implicit-check-not RANDOMIZE_REG
+// RUN: circt-opt %s -verify-diagnostics --pass-pipeline="builtin.module(lower-seq-to-sv{disable-reg-randomization})" | FileCheck %s --check-prefix DISABLED
```

Comment:
Consider:
```suggestion
// RUN: circt-opt %s -verify-diagnostics --pass-pipeline="builtin.module(lower-seq-to-sv{disable-reg-randomization})" | FileCheck %s --check-prefixes COMMON,DISABLED
```

To include the `-LABEL`'s?  This helps delimit the `-NOT`'s.

Nit:
As-is it looks like this has a sequence of (redundant) `-NOT` directives -- there's no positive check to separate them.
This could probably use `--implicit-check-not` to capture those directives but if having the explicit `DISABLED-NOT` inline serves a purpose that works too.

Reply:
I don't want to pass that much information on the command line, I think it's better to use `DISABLE-NOT` here.